grails-app/conf/au/org/emii/aatams/VisibilityControlFilters.groovy

```diff
@@ -36,15 +36,16 @@ class VisibilityControlFilters
         {
             after = { model ->
 
-                def instanceName = "${controllerName}Instance"
-                def instance = model[instanceName]
+                if (model) {
```

Comment:
I agree that this solution does not give any clue as to when/why the model might be null.